app/services/message-processor/check-addtional-crns.js

```diff
@@ -0,0 +1,24 @@
+const { checkContact } = require('../crm-client')
+
+const checkAdditionalCrns = async (listofCRNwithEmpowerment, validCrns, invalidCrns, logMessage, crn) => {
```

Comment:
I'm quite curious as to what the purpose of `logMessage` is? On lines 11-12 below I noticed that the ICE CRM uses both a `console.log` and a `logMessage` to essentially return the same message twice. I would assume this isn't duplication but rather that the `console.log` is for the application when running locally for example and that the `logMessage` is for CRM specifically? (It's because of this that I have opted to keep the `logMessage`).